Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35230&attempt_id=7d088c77-51d9-4ff3-b33d-7da9f22ea810)

<details><summary>Partial log (click to expand)</summary><p>

```
updateAutotoolsGnuConfigScriptsPhase
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/shjmvsnwrlg7rd8m0kzgpygqq4if312z-bash-4.4-p12/bin/bash
gcc -O3 -m64 -Wall -Wextra -Wno-unused-parameter -Werror -Iinc -o bcal src/log.c src/bcal.c -lquadmath
gcc: error: unrecognized command line option '-m64'
make: *** [Makefile:14: bcal] Error 1
builder for '/nix/store/s7sndyxciyg3j1w12jy9y8nsc9q12aal-bcal-1.7.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/s7sndyxciyg3j1w12jy9y8nsc9q12aal-bcal-1.7.drv' failed
```
</p></details>

--- a/pkgs/applications/science/math/bcal/default.nix
+++ b/pkgs/applications/science/math/bcal/default.nix
@@ -1,0 +1,33 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "bcal-${version}";
+  version = "1.7";
+
+  src = fetchFromGitHub {
+    owner = "jarun";
+    repo = "bcal";
+    rev = "v${version}";
+    sha256 = "08cqp2jysvy743gmwpzkbqhybsb49n65r63z3if53m3y59qg4aw8";
+  };
+
+  nativeBuildInputs = [ python3Packages.pytest ];
+
+  doCheck = true;
+  checkPhase = ''
+    python3 -m pytest test.py
+  '';
+
+  makeFlags = [ "CC=cc" ];
+  installFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
+
+  meta = {
+    description = "Storage conversion and expression calculator";
+    homepage = https://github.com/jarun/bcal;
+    license = licenses.gpl3;
+    platforms = [ "x86_64-darwin" "x86_64-linux" ];
+    maintainers = with maintainers; [ jfrankenau ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -19708,6 +19708,8 @@ with pkgs;
 
   pcalc = callPackage ../applications/science/math/pcalc { };
 
+  bcal = callPackage ../applications/science/math/bcal { };
+
   pspp = callPackage ../applications/science/math/pspp {
     inherit (gnome3) gtksourceview;
   };

